src/serpapi.ts

```diff
@@ -126,9 +129,12 @@ export async function getJson<
  * // callback
  * getHtml("google", { api_key: API_KEY, q: "coffee" }, console.log);
  */
-export async function getHtml<E extends keyof EngineMap>(
+export async function getHtml<
+  E extends EngineName = EngineName,
+  P extends EngineParameters<E> = EngineParameters<E>,
```

Comment:
Let's update `getHtml` too.

Reply:
Done in https://github.com/serpapi/serpapi-javascript/pull/8/commits/94b4dfea5145524cc1e9aff21aafbae678a908d9